TypeCobol/Compiler/Diagnostics/MessageCode.cs

```diff
@@ -62,6 +62,8 @@ public enum MessageCode
         InvalidExponentInDecimalFloatingPointLiteral = 51,
         InvalidMantissaInDecimalFloatingPointLiteral = 52,
         InvalidCharBeforePseudoTextDelimiter = 53,
-        ShouldBePrecededBySpace = 54
+        ShouldBePrecededBySpace = 54,
+        // End for SQL
```

Comment:
Well, actually `InvalidCharBeforePseudoTextDelimiter` and `ShouldBePrecededBySpace` are for Cobol but I did not add any comment to signal the end of SQL-related members. Could you move your comment up 2 lines ? Thanks.

Reply:
Done